src/main/scala/com/wavesplatform/features/FeatureStatus.scala

```diff
@@ -0,0 +1,13 @@
+package com.wavesplatform.features
+
+sealed case class FeatureStatus(status: Byte){
```

Comment:
I was able to create `FeatureStatus(5:Byte)` which makes no sense.
please rewrite to 
`sealed trait FeatureStatus`, `case class`es implementing and 
```
object FeatureStatus {
def apply(b: Byte) : Either[...,FeatureStatus]
}
```